plutus-contract/test/Spec/Contract/Tx/Constraints/RequiredSigner.hs

```diff
@@ -103,46 +89,46 @@ withoutOffChainMustBeSignedByContract submitTxFromConstraints paymentPubKey sign
             Constraints.collectFromTheScript utxos signedPubKey
             <> Constraints.mustIncludeDatumInTx unitDatum
     logInfo @String $ "Required Signatories: " ++ show (Constraints.requiredSignatories tx2)
-    ledgerTx2 <- submitTxFromConstraints lookups2 tx2
+    ledgerTx2 <- submitTxConstraintsWith lookups2 tx2
     awaitTxConfirmed $ Tx.getCardanoTxId ledgerTx2
 
-ownWallet :: SubmitTx -> TestTree
-ownWallet sub =
+ownWallet :: TestTree
+ownWallet =
     let trace = do
-            void $ Trace.activateContractWallet w1 $ mustBeSignedByContract sub w1PubKey w1PubKey
+            void $ Trace.activateContractWallet w1 $ mustBeSignedByContract w1PubKey w1PubKey
             void Trace.nextSlot
     in checkPredicateOptions defaultCheckOptions "own wallet's signature passes on-chain mustBeSignedBy validation" (assertValidatedTransactionCount 2) (void trace)
 
-otherWallet :: SubmitTx -> TestTree -- must use Trace.setSigningProcess for w2
-otherWallet sub =
+otherWallet :: TestTree -- must use Trace.setSigningProcess for w2
+otherWallet =
     let trace = do
             Trace.setSigningProcess w1 (Just $ signPrivateKeys [CW.paymentPrivateKey $ fromJust $ walletToMockWallet w1, CW.paymentPrivateKey $ fromJust $ walletToMockWallet w2])
-            void $ Trace.activateContractWallet w1 $ mustBeSignedByContract sub w2PubKey w2PubKey
+            void $ Trace.activateContractWallet w1 $ mustBeSignedByContract w2PubKey w2PubKey
             void Trace.nextSlot
     in checkPredicateOptions defaultCheckOptions "other wallet's signature passes on-chain mustBeSignedBy validation" (assertValidatedTransactionCount 2) (void trace)
 
-otherWalletNoSigningProcess :: SubmitTx -> TestTree
-otherWalletNoSigningProcess sub =
+otherWalletNoSigningProcess :: TestTree
+otherWalletNoSigningProcess =
     let trace = do
-            void $ Trace.activateContractWallet w1 $ mustBeSignedByContract sub w2PubKey w2PubKey
+            void $ Trace.activateContractWallet w1 $ mustBeSignedByContract w2PubKey w2PubKey
             void Trace.nextSlot
     in checkPredicateOptions defaultCheckOptions "without Trace.setSigningProcess fails phase-1 validation"
     (assertFailedTransaction (const $ cardanoLedgerErrorContaining "MissingRequiredSigners"))
     (void trace)
 
-withoutOffChainMustBeSignedBy :: SubmitTx -> TestTree -- there's no "required signer" in the txbody logs but still passes phase-2 so it must be there. Raised https://github.com/input-output-hk/plutus-apps/issues/645. It'd be good to check log output for expected required signer pubkey in these tests.
-withoutOffChainMustBeSignedBy sub =
+withoutOffChainMustBeSignedBy :: TestTree -- there's no "required signer" in the txbody logs but still passes phase-2 so it must be there. Raised https://github.com/input-output-hk/plutus-apps/issues/645. It'd be good to check log output for expected required signer pubkey in these tests.
```

Comment:
would you mind removing this comment and reference to issue because the expected outcome was fixed in an earlier PR